tests/CMakeLists.txt

```diff
@@ -246,3 +255,5 @@ foreach(DIP 1 2 3 4 5 6)
         endforeach()
     endforeach()
 endforeach()
+
+include(clean_tests.cmake)
```

Comment:
what is this file?

Reply:
its a test that 'cleans' the comparision tests. It was in the original `CMakeLists.txt` I'm not sure if I would end up keeping it or not